### DISCUSSION TITLE
importing modules from scipy suddenly started being reported as unknown symbols

### DISCUSSION THREAD

**Discussion by jasonboh** [microsoft/pyright#5427] [General]
I use VS Code with Pylance.  If this discussion doesn't belong here, please let me know.  I import scipy modules as shown below:

from scipy import optimize
from scipy import interpolate

In VS Code, if you hover your mouse over those modules, you can see that the modules are indeed able to be found by some part of the Python or Pylance VS Code extensions.  However, I believe that pyright is having problems finding these modules because they are reported as problems in the Problems pane.  These import statements previously were not flagged as problems by Pylance.  Is this a pyright issue?  Did something change in pyright recently to cause this?

**Comment by debonte** [comment]
You're hitting https://github.com/microsoft/pylance-release/issues/4531 which was fixed in Pylance's 2023.6.41 pre-release build from last week.
